core/src/main/java/hudson/model/JDK.java

```diff
@@ -7,8 +7,8 @@
  * of this software and associated documentation files (the "Software"), to deal
  * in the Software without restriction, including without limitation the rights
  * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- * copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
+ * copies, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
```

Comment:
Please don't change the license of existing source files.
```suggestion
 * copies of the Software, and to permit persons to whom the Software is
 * furnished to do so, subject to the following conditions:
```